### PR TITLE
HDDS-1522. Provide intellij runConfiguration for Ozone components

### PR DESCRIPTION
**Pull request by elek**
Sometimes I need to start ozone cluster from intellij to debug issues. It's possible but it requires to create many runConfiguration object inside my IDE.

I propose here to share the intellij specific runtimeConfigs to make it easy for anybody (who uses intellij) to run full ozone cluster from the IDE (1 datanode only).

See: https://issues.apache.org/jira/browse/HDDS-1522

### CODE REVIEW

File: hadoop-ozone/dev-support/intellij/install-runconfigs.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+SRC_DIR="$SCRIPT_DIR/runConfigurations"
+DEST_DIR="$SCRIPT_DIR/../../../.idea/runConfigurations/"
+#shellcheck disable=SC2010
+ls -1 "$SRC_DIR" | grep -v ozone-site.xml | xargs -n1 -I FILE cp "$SRC_DIR/FILE" "$DEST_DIR"
```
Comment (xiaoyuyao) [inline hadoop-ozone/dev-support/intellij/install-runconfigs.sh]: On my mac, I got the following errors when run the install-runconfig.sh
 
cp: directory /Users/XXX/git/hadoop/hadoop-ozone/dev-support/intellij/../../../.idea/runConfigurations does not exist

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 30 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 1 | No case conflicting files found. |
| 0 | shelldocs | 1 | Shelldocs was not available. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -1 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 425 | trunk passed |
| +1 | mvnsite | 0 | trunk passed |
| +1 | shadedclient | 798 | branch has no errors when building and testing our client artifacts. |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 409 | the patch passed |
| +1 | mvnsite | 0 | the patch passed |
| +1 | shellcheck | 0 | There were no new shellcheck issues. |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | xml | 14 | The patch has no ill-formed XML file. |
| +1 | shadedclient | 694 | patch has no errors when building and testing our client artifacts. |
||| _ Other Tests _ |
| +1 | unit | 91 | hadoop-hdds in the patch passed. |
| +1 | unit | 128 | hadoop-ozone in the patch passed. |
| +1 | asflicense | 37 | The patch does not generate ASF License warnings. |
| | | 2788 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-815/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/815 |
| Optional Tests | dupname asflicense mvnsite unit shellcheck shelldocs xml |
| uname | Linux 6e575e35faa9 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 5847e00 |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-815/1/testReport/ |
| Max. process+thread count | 410 (vs. ulimit of 5500) |
| modules | C: hadoop-ozone U: hadoop-ozone |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-815/1/console |
| versions | git=2.7.4 maven=3.3.9 shellcheck=0.4.6 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

File: hadoop-ozone/dev-support/intellij/install-runconfigs.sh (hunk 1 above)
Comment (elek) [reply hadoop-ozone/dev-support/intellij/install-runconfigs.sh]: Thanks the help, you are right An `mkdir -p` is missing. I will add it.

Comment (xiaoyuyao) [reply hadoop-ozone/dev-support/intellij/install-runconfigs.sh]: Yes, mkdir -p solves the script problem. However, it is not clear how to import these xml files into intelliJ. Can you provide some pointers on how to import these xml files and run them inside IntelliJ?

**Comment by anuengineer** [comment]
+1, LGTM. Merging now.

**Pull request merged**